test/machineslib.py

```diff
@@ -33,6 +34,8 @@ def hasMonolithicDaemon(image):
 
 
 class VirtualMachinesCaseHelpers:
+    machine: testvm.Machine
```

Comment:
What's this bit for?

Reply:
I've typed the new helper function and `self.machine` was not typed so mypy complains.

Reply:
Ah, this is a type, not a member declaration, my bad.